backend/src/controller/index.ts

```diff
@@ -1,8 +1,10 @@
 import express from 'express';
 import usersRoute from './user';
+import postRoute from './post';
```

Comment:
please align namings usersRoute but postRoute seems inconsistent and then routes are user and posts